app/helpers/versions_helper.rb

```diff
@@ -2,35 +2,80 @@ module VersionsHelper
   ##
   # Groups functions related to change description
   ##
-  def link_if_alive(version, link_text, link_url)
-    version.item ? link_to(link_text, link_url) : link_text
+  def link_if_alive(version, link_text, link_url, conference)
+    version.item && conference ? link_to(link_text, link_url) : "#{link_text} with ID #{version.item_id}"
+  end
+
+  def link_to_conference(conference_id)
+    return 'deleted conference' if conference_id.nil?
```

Comment:
We did not have conference_id attribute in versions since the beginning, that combined with records deleted directly from the DB can make it impossible for the conference_id attribute to be calculated.
I have 9 records (with event_type Splashpage, Venue, RegistrationPeriod) that don't have a conference_id. My DB comes from events.o.o which is where I pinpointed all those special cases.